core/mapi/cache/namedProps.cpp

```diff
@@ -237,7 +238,7 @@ namespace cache
 
 		if (lpProp && PT_BINARY == PROP_TYPE(lpProp->ulPropTag))
 		{
-			const auto bin = mapi::getBin(lpProp);
+			const auto &bin = mapi::getBin(lpProp);
```

Comment:
This was based on some intellisense feedback. I think getBin must return a reference and this avoids a copy.